### PR TITLE
Implement the <aside>

### LINKED ISSUE #3: Implement the <aside>

### PR DESCRIPTION
**Pull request by damdiana**

### CODE REVIEW

File: buttons.css [hunk 1]
```diff
@@ -52,4 +52,22 @@
...
+
+.filter-question .btn:first-child {
+    border-bottom-left-radius: 0.2em;
+    border-top-left-radius: 0.2em;
+}
+
+.filter-question .btn:nth-last-child(2) {
+    border-bottom-right-radius: 0.2em;
+    border-top-right-radius: 0.2em;
+}
```
Comment (iampava) [inline buttons.css]: You already have these styles in `style.css`. Delete from here

File: inputs.css [hunk 2]
```diff
@@ -16,7 +16,7 @@
     position: absolute;
     font-size: inherit;
     fill: var(--gray-2);
-    left: 0.5em;
+    left: 2.5em;
```
Comment (iampava) [inline inputs.css:19]: Why change this here? This PR shouldn't be worries about any inputs, right?

File: style.css [hunk 3]
```diff
@@ -101,10 +101,107 @@ nav ul {
     line-height: 2em;
 }
 
+/* main */
+
+main {
+    width: 60%;
+}
```
Comment (iampava) [inline style.css]: This should have been a "red flag". You already set up a width for main in the other PR. Delete this one.

Also, look in `style.css`, you have this rule 3 times in total. Keep just one.

File: style.css [hunk 4]
```diff
@@ -101,10 +101,107 @@ nav ul {
...
+
+h1 {
+    font-size: 25px;
+    font-weight: normal;
+}
```
Comment (iampava) [inline style.css]: This already exists. Delete.

File: style.css [hunk 5]
```diff
@@ -101,10 +101,107 @@ nav ul {
...
+
+.summary {
+    width: 10%;
+    color: var(--gray-2);
+    flex-shrink: 0;
+    font-size: 14px;
+    padding-left: 3em;
+}
```
Comment (iampava) [inline style.css]: This rule already exists in the file. Delete.

File: style.css [hunk 6]
```diff
@@ -101,10 +101,107 @@ nav ul {
...
+
+h2 {
+    font-size: 20px;
+    font-weight: normal;
+    color: var(--blue);
+    margin: 0;
+}
```
Comment (iampava) [inline style.css]: These already exist. Delete.

File: style.css [hunk 7]
```diff
@@ -113,14 +210,16 @@ nav ul {
...
 
-.text-with-icon svg {
-    position: absolute;
-    width: 1em;
-    fill: var(--gray-2);
+.icon-top {
+    top: 0;
+}
+
+.icon-center svg {
```
Comment (iampava) [inline style.css]: Let's use BEM naming from the class and rename to `text-with-icon--center`. This way we know we have a "center" modifier for the `text-with-icon`.

File: style.css [hunk 8]
```diff
@@ -113,14 +210,16 @@ nav ul {
     right: 1em;
 }
 
-.text-with-icon svg {
-    position: absolute;
-    width: 1em;
-    fill: var(--gray-2);
+.icon-top {
```
Comment (iampava) [inline style.css]: Let's use BEM naming from the class and rename to `text-with-icon--top`. This way we know we have a "top" modifier for the text-with-icon.

File: style.css [hunk 9]
```diff
@@ -101,10 +101,107 @@ nav ul {
...
+
+.btn-icon.gray svg {
+    fill: var(--gray-2);
+}
+
+.btn-icon.navy svg {
+    fill: var(--navy);
+}
+
+aside .text-with-icon.icon-top svg {
```
Comment (iampava) [inline style.css]: Why do we care if it's inside an `aside`? The text with icon class/styles should work the same no matter where we put them.

File: style.css [hunk 10]
```diff
@@ -101,10 +101,107 @@ nav ul {
...
+    height: 1em;
+}
+
+.btn-icon.gray svg {
+    fill: var(--gray-2);
+}
+
+.btn-icon.navy svg {
+    fill: var(--navy);
+}
```
Comment (iampava) [inline style.css]: These should be in the `buttons.css` file.

File: style.css [hunk 11]
```diff
@@ -101,10 +101,107 @@ nav ul {
...
+
+.post-summary {
+    gap: 1em;
+    margin: 0;
+    padding: 0;
+}
+
+.post-summary a {
+    color: var(--blue);
+}
```
Comment (iampava) [inline style.css]: I think all these rules already exist. Delete

**Review by iampava — CHANGES_REQUESTED** [review]
Pay more attention when fixing conflicts.  You have a lot of duplicate code.

File: inputs.css (hunk 2 above)
Comment (damdiana) [reply inputs.css:19]: I checked the main branch and it's a bug. I fixed it in this commit.

File: style.css [hunk 12]
```diff
@@ -175,10 +145,81 @@ h2 a:visited {
...
+    fill: currentColor;
+}
+
+.text-with-icon.icon-left svg {
+    left: -1.40em;
+}
+
+.text-with-icon.icon-right svg {
+    right: 1em;
+}
```
Comment (iampava) [inline style.css]: Please follow the convention everywhere `text-with-icon--VARIANT`

File: style.css [hunk 13]
```diff
@@ -175,10 +145,81 @@ h2 a:visited {
...
+    padding-top: 0;
+    width: 100%;
+    background-color: hsl(46deg 87% 91%);
+    border: 1px solid hsl(46deg 65% 84%);
+    padding: 0.5em;
+}
+
+/* icons */
+
+.text-with-icon.text-with-icon--top svg {
```
Comment (iampava) [inline style.css]: This rule already exists on line `213`. Also, why do we have a `width` here? The `top` variant shouldn't care about widths.

File: style.css [hunk 14]
```diff
@@ -175,10 +145,81 @@ h2 a:visited {
...
+.filter-question .btn:first-child {
+    border-bottom-left-radius: 0.2em;
+    border-top-left-radius: 0.2em;
+}
+
+.filter-question .btn:nth-last-child(2) {
+    border-bottom-right-radius: 0.2em;
+    border-top-right-radius: 0.2em;
+}
+
```
Comment (iampava) [inline style.css]: Previously this code was on line `135`. You moved it down, and now looks like changed (see picture)
image

Please put it back to it;s place, so we don't see it at "files changed"

**Review by iampava — CHANGES_REQUESTED** [review]
A few more problems.

**Review by iampava — APPROVED** [review]

**Pull request merged**